docs/BackupRestoreGuide.md

```diff
@@ -0,0 +1,43 @@
+# Backup restore guide for Scalar products
```

Comment:
Following the guidance in [Google Developer Documentation Style Guide ("Heading and title text")](https://developers.google.com/style/headings#heading-and-title-text), how about we use the following?:

```suggestion
# Back up and restore ScalarDB or ScalarDL data in a Kubernetes environment